webrender_traits/src/types.rs

```diff
@@ -371,10 +372,13 @@ pub enum MixBlendMode {
 pub type NativeFontHandle = CGFont;
 
 /// Native fonts are not used on Linux; all fonts are raw.
-#[cfg(not(target_os = "macos"))]
-#[cfg_attr(not(target_os = "macos"), derive(Clone, Serialize, Deserialize))]
+#[cfg(all(unix, not(target_os="macos")))]
```

Comment:
I don't understand what that change is supposed to do, but please do try not(windows or mac).